wpeverest/everest-forms
# Dev - Subscription plan field with trail period

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces the new subscription plan field for PayPal and Stripe integration. Now we can create multiple subscription plan with trial period.

### How to test the changes in this Pull Request:

1. Setup the payments and edit a form
2. Drag the subscription plan field choose trail periods amount etc
3. Enable the recurring payment from the form settings > payments > also choose the email for notify.
4. Submit form and check with and without trial.
5. Also check the backward compatility.

### Types of changes:


### Other information:


### Changelog entry

> Dev - Subscription plan field with trail period.
